### PR TITLE
[13.0][IMP]remove unnecessary depends from l10n_be_partner_kbo_bce

### PR DESCRIPTION
**Pull request by luc-demeyer**

### CODE REVIEW

File: l10n_be_partner_kbo_bce/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "license": "AGPL-3",
     "author": "Noviat,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/l10n-belgium",
-    "depends": ["partner_identification", "base_vat_sanitized"],
+    "depends": ["partner_identification"],
```
Comment (sbidoul) [inline l10n_be_partner_kbo_bce/__manifest__.py]: It seems `_fix_vat_number` is in `base_vat`. Should it not depend on it ? @luc-demeyer -de

**Comment by luc-demeyer** [comment]
@sbidoul 
can you merge ?

**Comment by sbidoul** [comment]
sure. /ocabot merge

**Pull request merged**